packages/form-js-viewer/assets/form-js.css

```
@@ -483,6 +487,10 @@
   padding: 2px 6px 2px 8px;
 }
 
+.fjs-container .fjs-taglist .fjs-taglist-tag.disabled .fjs-taglist-tag-label {
+  padding: 2px 8px 2px 8px;
```

Can be shortened a bit

```suggestion
  padding: 2px 8px;
```